### ISSUE TITLE
[Visual requirement – Make Code – Create a project] Luminosity ratio is less than 3:1 for focus indicator over the 'Close' button and border line of 'Give your project name' edit field.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3556]
**User Experience:**
Visual impaired user will find difficulty to see the focus indicator over the control or element and border line for edit field if that focus indicator and border line have insufficient color contrast ratio as user won't be able to proceed further.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to the controls appearing under "My Project" section in Home page and select 'New Project' Button.
4. Navigate all controls under 'Create a Project' pop-up.
5. Verify contrast ratio of focus indicator over the 'Close' button and border line of 'Give your project name' edit field.

**Actual Result:**
Luminosity ratio is less than 3:1 for focus indicator over the 'Close' button and border line of 'Give your project name' edit field under 'Create a project' pop-up and due to this, visual impaired user is unable to see the focus indicator and border line.

**Expected Result:**
Luminosity ratio should be more than and equal to 3:1 for focus indicator over the 'Close' button and border line of 'Give your project name' edit field under 'Create a project' pop-up, so that visual impaired user could easily see the border of focus.

**Note:**
Same issue exists on Firefox & Chrome.
Same issue exists throughout the application where this type of control appears.

**WCAG Reference:**
[WCAG 1.4.11 - Non-text Contrast
](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Tool: CCA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG1.4.11, #Win10-Edge(Chromium)

![WCAG1 4 11_Create a project_CCA](https://user-images.githubusercontent.com/72871089/97955089-58c7fc80-1dcb-11eb-9eba-c9b518a3e5e3.jpg)
